src/@apollo/client/link/core/ApolloClient__Link_Core_Types.re

```diff
@@ -54,19 +54,35 @@ module Operation = {
     variables: Js.Json.t,
     operationName: string,
     extensions: Js.Json.t,
+    [@bs.as "reason_setContext"]
     setContext: Js.Json.t => Js.Json.t,
+    [@bs.as "reason_getContext"]
     getContext: unit => Js.Json.t,
   };
 
+  let preserveJsPropsAndContext: (Js_.t, t) => t = [%bs.raw
```

Comment:
I feel like there must be a better way to do this, but not worrying about it right now.